examples/analytics-notification.amp.html

```diff
@@ -66,8 +66,8 @@
   <amp-user-notification
       layout=nodisplay
       id="amp-user-notification1"
-      data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
It worked via localhost, but I did get an error when accessing via http://dinghongfei.mtv.corp.google.com. 

`source must start with "https://" or "//" or be relative and served from either https or from localhost. Invalid value: /api/show?timestamp=TIMESTAMP​​​ _reported_`

Why don't we allow relative URLs over HTTP? I don't see it mentioned in the [amp-user-notification doc](https://github.com/ampproject/amphtml/blob/master/extensions/amp-user-notification/amp-user-notification.md). Is it something general we tend to disallow in the whole AMP project? If so, could you point me to a reference?